transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/util/ImportExportPackageUpdater.java

```diff
@@ -199,7 +269,7 @@ private String addELPackage(String importPkgs) {
 			importPkgs = "";
 		}
 		if (!importPkgs.isEmpty()){
-			importPkgs += ",\n"; //$NON-NLS-1$
+			importPkgs += ",\n "; //$NON-NLS-1$

```

Comment:
- will we have one package which will be longer than the max line width?
- During Jar generation the manifest is not reformatted? (it is the case for Tycho but i'm not familiar with other way of doing it)